monkey/infection_monkey/tunnel.py

```diff
@@ -195,5 +204,17 @@ def get_tunnel_for_ip(self, ip: str):
         ip_match = get_interface_to_target(ip)
         return "%s:%d" % (ip_match, self.local_port)
 
+    def report_exploited_machine(self):
```

Comment:
this method doesn't report anything. I'd say a better name would be `on_successful_exploitation` to indicate that it needs to be called when machine got successfully exploited?

Reply:
`increment_exploited_machine_counter()`?
`mark_machines_exploited()`?
`set_machine_successfully_exploited()`?